client/rest-high-level/src/test/java/org/elasticsearch/client/BulkProcessorIT.java

```diff
@@ -212,16 +212,15 @@ public void testBulkProcessorWaitOnClose() throws Exception {
 
     public void testBulkProcessorConcurrentRequestsReadOnlyIndex() throws Exception {
         Request request = new Request("PUT", "/test-ro");
-        request.setJsonEntity(
-            "{\n"
-                + "    \"settings\" : {\n"
-                + "        \"index\" : {\n"
-                + "            \"blocks.write\" : true\n"
-                + "        }\n"
-                + "    }\n"
-                + "    \n"
-                + "}"
-        );
+        request.setJsonEntity("""
+            {
+                "settings" : {
+                    "index" : {
+                        "blocks.write" : true
+                    }
+                }
+               \s
```

Comment:
That looks peculiar - we could just remove this line?

Reply:
Yeah, the PR is still very WIP. I'm going to go over all the strings manually and fix peculiarities like that.